src/mono/wasm/runtime/library_mono.js

```diff
@@ -196,6 +222,11 @@ var MonoSupportLib = {
 		},
 
 		_mono_wasm_root_buffer_prototype: {
+			__handle: 0,
```

Comment:
Is this necessary for strict mode? I think it changes performance semantics

Reply:
Hmm, thank you for pointing it out. Strict mode does not allow using undeclared variables. However, I guess this is an exception since it is part of an object. I just tried without these declarations and it works. I'll push a fix to remove these declarations